dftimewolf/lib/utils.py

```diff
@@ -17,6 +18,11 @@
 TOKEN_REGEX = re.compile(r'\@([\w_]+)')
 
 
+def CalculateRunTime(time_start: float) -> float:
+  """Calculates a time delta used for runtime calulcations."""
```

Comment:
Given all the operations happening, it would be nice to specify which units we're taking in and returnin (e.g. seconds, milliseconds, etc.)